local-volume/provisioner/pkg/util/api_util.go

```diff
@@ -97,6 +101,10 @@ func (u *apiUtil) DeleteJob(jobName string, namespace string) error {
 	return nil
 }
 
+func (u *apiUtil) GetStorageClass(scName string) (*storage_v1.StorageClass, error) {
+	return u.client.StorageV1().StorageClasses().Get(scName, metav1.GetOptions{})
```

Comment:
The RBAC permissions need to add StorageClass Get permissions

Reply:
The `system:persistent-volume-provisioner` clusterrole [we are using](https://github.com/kubernetes-incubator/external-storage/blob/master/local-volume/helm/provisioner/templates/provisioner-cluster-role-binding.yaml#L13) contains `get` permissions on `storageclasses`, no need to add.

Reply:
Can we use informer to get StorageClass objects? To reduce chance of exhausting API quota.

Reply:
@msau42 Updated in https://github.com/kubernetes-incubator/external-storage/pull/776/commits/346c2e28611b81616f4ad21f36913d7228e00712.

- start StorageClass informer controller in Discoverer, then it can get StorageClass from informer cache
- change `common.RuntimeConfig.Client` to be `kubernetes.Interface`, then we can use `fake.Clientset` for testing
- update `Populator.Start` to be same as `Discoverer.Start` which are almost the same
- prefer using `wait.NeverStop` which is more idiomatic in k8s